docs/quickstarts/aws.md

```diff
@@ -5,174 +5,218 @@ weight: 10
 
 ## Introduction
 
-This quickstart guide walks through the steps needed to create a Lokomotive cluster on AWS with
-Flatcar Container Linux using Route53 as the DNS provider.
+This quickstart guide walks through the steps needed to create a Lokomotive cluster on AWS.
+
+Lokomotive runs on top of [Flatcar Container Linux](https://www.flatcar-linux.org/). This guide
+uses the `stable` channel.
+
+The guide uses [Amazon Route 53](https://aws.amazon.com/route53/) as a DNS provider. For more
+information on how Lokomotive handles DNS, refer to [this](../concepts/dns.md) document.
+
+Lokomotive stores Terraform state [locally](../configuration-reference/backend/local.md) by default,
+or remotely within an [AWS S3 bucket](../configuration-reference/backend/s3.md).
+
+[Lokomotive components](../concepts/components.md) complement the "stock" Kubernetes functionality
+by adding features such as load balancing, persistent storage and monitoring to a cluster. To keep
+this guide short you will deploy a single component - `httpbin` - which serves as a demo
+application to verify the cluster behaves as expected.
 
 By the end of this guide, you'll have a production-ready Kubernetes cluster running on AWS.
 
 ## Requirements
 
 * Basic understanding of Kubernetes concepts.
-* AWS account and IAM credentials.
-* AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
+* An AWS account and IAM credentials.
+* An AWS
+  [access key ID and secret](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
+  of a user with
```

Comment:
Alternatively it can be changed to `An AWS access key ID and a user's secret with permissions to edit Route 53 records.`.